### DISCUSSION TITLE
Does Drag and Drop work after patched explorer.exe ?

### DISCUSSION THREAD

**Discussion by popy2k14** [valinet/ExplorerPatcher#70] [Q&A] [answered]
Dear valinet.

Thanks a lot for bringing back some usabillity to Windows 11 :-)
Until now, i have not updated to Win11 but following your project.

One question: Is Drag and Drop to Taskbar also supported after applying your patches?

thx

**Comment by Gaurav-Original-ClassicShellTester** [comment, marked as answer]
Of course it does.

**Reply by JoePalffy** [reply]
So is the answer still "Of course it does." in 2025?
because I can't find such an option in your settings and neither does drag and drop work anymore after Win 11 23h2 reinstalled itself on my PC.

**Reply by pyrates999** [reply]
why would there need to be an option for this?

**Reply by JoePalffy** [reply]
to see if it's included, 
also an older Microsoft answers reply said or at least hinted at there being an option called "Enhance Drag and Drop" or something like that, either way it doesn't work and that's what matters.

**Comment by popy2k14** [comment]
thx a lot for confirming!

**Comment by moenkaey** [comment]
Can one get a specific option to enable drag and drop for windows 11 taskbar ?

**Reply by valinet** [reply]
Maybe have a look at: [Windows 11 Drag & Drop to the Taskbar (Fix)](https://github.com/HerMajestyDrMona/Windows11DragAndDropToTaskbarFix).
